packages/socket/test/socket_spec.js

```diff
@@ -32,10 +34,10 @@ describe('Socket', function () {
   })
 
   context('.getClientSource', function () {
-    it('returns client source as a string', function(done) {
-      const p = process.cwd() + '/node_modules/socket.io-client/dist/socket.io.js'
+    it('returns client source as a string', function (done) {
+      const p = `${process.cwd()}/node_modules/socket.io-client/dist/socket.io.js`
```

Comment:
I think we should be more consistent about using `path.join` here

Reply:
Agreed. I think the only reason this changed is that eslint auto-fixed it.